azure-pipelines.yml

```diff
@@ -30,13 +30,13 @@ stages:
 
       dependencies:
         - source: '\\nirvana\Measurements\VeriStandAddons\slsc_switch_messaging_library'
-          file: 'CDMessaging.lvlibp'
+          file: '$target\CDMessaging.lvlibp'
```

Comment:
Will not work until https://github.com/ni/niveristand-custom-device-build-tools/pull/163 is submitted.

Reply:
merged